docs/excel/excel-add-ins-ranges-advanced.md

```diff
@@ -0,0 +1,163 @@
+Skipped cells will not overwrite the existing data of their corresponding cells in the destination range. The default is false.
+
+The following code sample and images demonstrate this behavior in a simple scenario. 
+
+```js
+Excel.run(function (context) {
+    var sheet = context.workbook.worksheets.getItem("Sample");
+    // copy a range, omitting the blank cells so existing data is not overwritten in those cells
+    sheet.getRange("D1").copyFrom("A1:C1",
+        Excel.RangeCopyType.all,
+        true, // skipBlanks
+        false); // transpose
+    // copy a range, including the blank cells which will overwrite existing data in the target cells
+    sheet.getRange("D2").copyFrom("A2:C2",
+        Excel.RangeCopyType.all,
+        false, // skipBlanks
+        false); // transpose
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+*Before the preceding function has been run.*
+
+![Data in Excel before range’s copy method has been run](../images/excel-range-copyfrom-skipblanks-before.png)
+
+*After the preceding function has been run.*
+
+![Data in Excel after range’s copy method has been run](../images/excel-range-copyfrom-skipblanks-after.png)
+
+`transpose` determines whether or not the data is transposed, meaning its rows and columns are switched, into the source location.
+A transposed range is flipped along the main diagonal, so rows **1**, **2**, and **3** will become columns **A**, **B**, and **C**.
+
+## Remove duplicates
+
+> [!NOTE]
+> The `Range.removeDuplicates` function is currently available only in public preview (beta). To use this feature, you must use the beta library of the Office.js CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js.
+> If you are using TypeScript or your code editor uses TypeScript type definition files for IntelliSense, use https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts.
+
+`Range.removeDuplicates` removes rows with duplicate entries in the specified columns. The function goes through each row in the range from top (the lowest index row in the range) to bottom (the highest valued index in the range). If a value in the specified column or columns is encountered more than once, that value's row is deleted. Subsequent rows in the range are shifted up. `removeDuplicates` does not affect the position of cells outside of the range.
+
+`Range.removeDuplicates` takes in a `number[]` representing the column indices which are checked for duplicates. This array is zero-based and relative to the range, not the worksheet. The function also takes in a boolean argument, `includesHeader` to determine if the range's top row is a header. When **true**, the top row is ignored when considering duplicates. A `RemoveDuplicatesResult` is returned upon completion of `removeDuplicates`, containing the number of rows removed and the number of unique rows remaining.
```

Comment:
Since you don't explicitly name the "number[]" parameter, I'd suggest you don't do so for the boolean parameter either (and you should use the word "parameter" for both, not "parameter" for one and "argument" for the other).

Consider rephrasing: 
"The function also takes in a boolean argument, `includesHeader` to determine if the range's top row is a header." 
-> 
"The function also takes in a boolean parameter that specifies whether the first row in the range is a header."